src/stylesheets/mixins.less

```diff
@@ -152,3 +152,15 @@
 .loader-style(@color: @impac-widget-loading-spinner) {
   color: @color;
 }
+
+.column-gutter(@gutter: @grid-gutter-width) {
```

Comment:
careful with that: are you sure it won't override the bootstrap class on all the maestrano's dashboard?
check if you can use @impac-padding-between-widgets instead